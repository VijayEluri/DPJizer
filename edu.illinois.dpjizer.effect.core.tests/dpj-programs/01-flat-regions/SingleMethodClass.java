/**
 * This file is licensed under the University of Illinois/NCSA Open Source License. See LICENSE.TXT for details.
 */
package flatregion.singlemethodclass;

import java.util.Date;

/**
 * 
 * @author Mohsen Vakilian
 * @author Danny Dig
 *
 */
class SingleMethodClass {
	region r1;
	Date f1 in r1;

	SingleMethodClass() {}
	
	void m1(Date x) /*writes r1*/ {
		f1 = x;
	}
}
