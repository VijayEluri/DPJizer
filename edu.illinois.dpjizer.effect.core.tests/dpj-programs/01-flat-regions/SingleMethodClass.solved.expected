flatregion.singlemethodclass.SingleMethodClass.SingleMethodClass() pure
flatregion.singlemethodclass.SingleMethodClass.m1(java.util.Date) writes Root : flatregion.singlemethodclass.SingleMethodClass.r1
